README.rst

```diff
@@ -136,25 +131,24 @@ The following code implements a simple dual moving average algorithm.
             return
 
         # Compute averages
-        # history() has to be called with the same params
+        # data.history() has to be called with the same params
         # from above and returns a pandas dataframe.
-        short_mavg = history(100, '1d', 'price').mean()
-        long_mavg = history(300, '1d', 'price').mean()
-
-        sym = symbol('AAPL')
```

Comment:
I had changed it to look like that thinking it would definitely be faster, but also because we use similar syntax in our tutorial series; given that most of the code written in the Quantopian IDE would be similar to an algo written in zipline, I tried to make the two examples (`buyapple.py` and the tutorial example) match up